heka/meta/heka.yml

```diff
@@ -375,6 +375,39 @@ aggregator:
           relational_operator: '=='
           threshold: 100
     - status: unknown
+  # A policy that is typically used for storage or compute clusters
+    majority_of_node_members:
+    - status: down
+      trigger:
+        logical_operator: or
+        rules:
+        - function: percent
+          arguments: [ down ]
+          relational_operator: '=='
+          threshold: 100
+    - status: critical
+      trigger:
+        logical_operator: and
+        rules:
+        - function: percent
+          arguments: [ down, critical ]
+          relational_operator: '>'
+          threshold: 50
+        - function: percent
+          arguments: [ okay ]
+          relational_operator: '<='
+          threshold: 50
+          function: percent
```

Comment:
Note: if more than 50% of members are down or critical then it is not possible to also have more than 50% of members in the okay state, so I think the second rule is not necessary.